<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbManagerDlgPublishTable</class>
 <widget class="QDialog" name="DbManagerDlgPublishTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Publish table to database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Output table</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Database</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radCreateNewTable">
        <property name="text">
         <string>Create new table</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Schema</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboSchema">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Table</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cboTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboDatabase"/>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radUseExistingTable">
        <property name="text">
         <string>Use existing table</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineTableName"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Table</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>cboTable</zorder>
     <zorder>label_2</zorder>
     <zorder>cboSchema</zorder>
     <zorder>label_3</zorder>
     <zorder>cboDatabase</zorder>
     <zorder>label</zorder>
     <zorder>radUseExistingTable</zorder>
     <zorder>radCreateNewTable</zorder>
     <zorder>lineTableName</zorder>
     <zorder>label_4</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="radSynchronize">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Synchronize changes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radCreate">
        <property name="text">
         <string>Replace all data</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkDropTable">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Drop existing one</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSpatialIndex">
        <property name="text">
         <string>Create spatial index</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <tabstops>
  <tabstop>radCreate</tabstop>
  <tabstop>chkDropTable</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbManagerDlgPublishTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radCreate</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkDropTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbManagerDlgPublishTable</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
